### PR TITLE
Quality of life shit

### PR DESCRIPTION
**Pull request by James-Discord**
james doesnt know how to code :(

### CODE REVIEW

File: src/commands/server/create.js [hunk 1]
```diff
@@ -13,10 +13,10 @@ module.exports = {
     requiredPermissions: [],
     checks: [{
         check: () => config.discord.commands.serverCommandsEnabled,
-        error: "The server commands are disabled!"
+        error: "The commands for this server are disabled!"
```
Comment (Darker-Ink) [inline src/commands/server/create.js:16]: The command category is server thats why its like that

**Review by Darker-Ink — CHANGES_REQUESTED** [review]

File: src/commands/server/create.js (hunk 1 above)
Comment (Darker-Ink) [reply src/commands/server/create.js:16]: @James-Discord Fix this and I'll merge it

**Review by Darker-Ink — CHANGES_REQUESTED** [review]

File: src/commands/server/create.js (hunk 1 above)
Comment (Darker-Ink) [reply src/commands/server/create.js:16]: @James-Discord Fix this and I'll merge it

**Comment by Darker-Ink** [comment]
Closing since been ignored

**Review by James-Discord — COMMENTED** [review]
g

**Pull request closed without merging**